graphql2/graphqlapp/rotation.go

```diff
@@ -375,3 +375,36 @@ func (m *Mutation) UpdateRotation(ctx context.Context, input graphql2.UpdateRota
 	}
 	return true, nil
 }
+
+func (a *Query) NextRotationHandoffTimes(ctx context.Context, input *graphql2.NextRotationHandoffTimesInput) ([]time.Time, error) {
+	var result []time.Time
+	var err error
+
+	err = validate.Many(
+		err,
+		validate.Range("input.count", input.Count, 0, 20),
+		validate.Range("input.hours", input.Hours, 0, 99999),
+	)
+	if err != nil {
+		return result, err
+	}
+
+	loc, err := util.LoadLocation(input.TimeZone)
+	if err != nil {
+		return result, errors.Wrap(err, "failed to load location for "+input.TimeZone)
```

Comment:
Errors should be more direct like `load tz location: not found`. However in this case, the wrapper is redundant as `LoadLocation` will return the name if relevant already (e.g., `unknown time zone asdf`).

Additionally, this should be returned as a field validation error.